master/pkg/model/model.go

```diff
@@ -4,36 +4,39 @@ import (
 	"fmt"
 	"strings"
 	"time"
+
+	"github.com/google/uuid"
 )
 
 // Model represents a row from the `models` table.
 type Model struct {
-	ID              int       `db:"id" json:"id"`
```

Comment:
I reordered fields to match the postgres output. I can keep the ordering consistent so that changelog is easier to follow if that's preferrable.